apps/api-extractor/src/generators/StringWriter.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { StringBuilder } from '@microsoft/tsdoc';
+
+// A small helper used by the generators
+export class StringWriter {
+  public readonly stringBuilder: StringBuilder = new StringBuilder();
+
+  public write(s: string): void {
+    this.stringBuilder.append(s);
+  }
+
+  public writeLine(s: string = ''): void {
```

Comment:
This is a little similar to the `Terminal` API. Should this make use of it?

Reply:
The `StringWriter` class is somewhat half-baked. Long term I will probably replace it with something with more thought put into it. The purpose of this PR was merely to make it easier to share some logic between ReviewFileGenerator and DtsRollupGenerator, since one was using `StringWriter ` but the other was using `StringBuilder` without any good reason for being different.